Riot/Modules/Room/RoomViewController.m

```diff
@@ -5096,17 +5096,37 @@ - (void)showContextualMenuForEvent:(MXEvent*)event cell:(id<MXKCellRendering>)ce
                                                                        completion:^{
                                                                            [self contextualMenuAnimationCompletionAfterBeingShown:YES];
                                                                        }];
-
+    
     if (RiotSettings.shared.messageReaction && [cell isKindOfClass:MXKRoomBubbleTableViewCell.class])
     {
-        // Note: For the moment, we use the frame of the cell instead of the component
-        // From UI perpective, that means the menu will be around a paragraph instead of a message
-        // This is not bad as the paragraph is kept visible to provide more context to the user
-        // TO FIX: if paragraph is bigger than the screen, the menu is displayed in the middle
-        CGRect frame = ((MXKRoomBubbleTableViewCell*)cell).frame;
-        frame = [self.bubblesTableView convertRect:frame toView:[self.bubblesTableView superview]];
-
-        [self.roomContextualMenuPresenter showReactionsMenuForEvent:event.eventId inRoom:event.roomId session:self.mainSession aroundFrame:frame];
+        MXKRoomBubbleTableViewCell *roomBubbleTableViewCell = (MXKRoomBubbleTableViewCell*)cell;
```

Comment:
That may worth to check the type before casting

Reply:
Check was already done above `if (RiotSettings.shared.messageReaction && [cell isKindOfClass:MXKRoomBubbleTableViewCell.class])`

Reply:
My bad. Sorry

Reply:
My bad. Sorry